src/app/person-search/components/person-search-header/person-search-header.component.html

```diff
@@ -1,10 +1,13 @@
 <div>
   <div class="pt-3 ps-3 pe-3">
+    <p class="mt-3">{{ 'person-search.name' | translate }}</p>
     <erz-typeahead
       [typeaheadService]="studentsRestService"
       [value]="filter.student"
       (valueChange)="$event && navigateToDossier($event)"
     >
     </erz-typeahead>
+    <hr />
+    <p class="mt-3">{{ 'person-search.description' | translate }}</p>
```

Comment:
Zur Trennlinie:

Ich würde mich an den Style halten wie wir diese Trennlinien auch sonst in der App machen (und wie es übrigens auch im Mockup skizziert ist). Du kannst z.B. bei `edit-absences-header` abschauen...

D.h. dem ganzen `<div>` hier ein `border-bottom: 1px solid $border-color;` geben und das ` <p class="mt-3">{{ 'person-search.description' | translate }}</p>` als Platzhalter anzeigen, wenn keine Resultate vorhanden sind (also hier aus dem Header rausnehmen und dort rendern wo die Resultate gerendered werden).

Zum Label:

Auch beim Label kannst du bei `edit-absences-header` oder so abschauen. Du solltest in einem solchen Fall unbedingt ein `<label>` Tag verwenden, idealerweise mit einem `for="id-des-inputs"` Attribut, damit man auch auf das Label klicken kann um den Fokus zu aktivieren (das haben wir leider nicht immer gemacht). Zudem schnallen dies Screenreaders so und können das Feld beschreiben.

Reply:
:+1: